vtex/faststore
# chore(gatsby-theme-store): Remove gatsby-config.js from theme-store

## What's the purpose of this pull request?
As title says, this PR removes gatsby-config from theme-store, virtually making it a plugin. The idea being that we can use different styling systems on different stores.

## How to test this PR?
This is also an architectural change and should not impact on the user's final experience. Make sure this is the case for:

https://github.com/vtex-sites/btglobal.store/pull/784
https://github.com/vtex-sites/marinbrasil.store/pull/599
https://github.com/vtex-sites/storecomponents.store/pull/1108
